Src/AutoFoqUnitTest/FoqMethodQueryTest.fs

```diff
@@ -87,6 +105,17 @@ let SelectMethodsDefineCorrectParameters (request: Type) =
                        (resultParameters |> Seq.toArray))) @>
     // Teardown
 
+[<Fact>]
+let ``Builder is correct`` () =
+    let expected =
+        { new ISpecimenBuilder with
+              member this.Create(r, c) = obj() }
+    let sut = FoqMethodQuery expected
+
+    let actual = sut.Builder
+
+    verify <@ expected = actual @>

```

Comment:
Added 3a5e842, in order to address [this](https://github.com/AutoFixture/AutoFixture/pull/464#discussion_r44374058) comment of mine. – Done adding commits.